README.md

```diff
@@ -10,6 +10,47 @@ The site is published automatically by [Netlify](https://netlify.com) whenever c
 
 To run the site locally, you'll need to install [Yarn](https://yarnpkg.com) and [Hugo](https://gohugo.io) (in particular the [extended](https://gohugo.io/getting-started/installing/) version).
 
+You can alternatively use the provided `Dockerfile` via:
+
+```bash
+$ make docker-image
+$ make docker-serve
+```
+
+If the locally website is running, the below message should be show.
+```markdown
+docker run --rm --interactive --tty --volume `pwd`:/home/builder/build -p 13131:13131 tikv-website sh -c "yarn && make serve-production"
+yarn install v1.16.0
```

Comment:
Maybe we can ask them to look for "Web Server is available at http://localhost:1313/ (bind address 0.0.0.0)"?